photo_points/Controllers/AdminController.cs

```diff
@@ -52,27 +59,40 @@ public IActionResult PhotoStream()
         [HttpGet]
         public IActionResult Pending()
         {
-            return View("Pending");
-            //all the new submissions can be displayed
-            // //  if Capture approval is false. 
-            // // display the Capture image and Data. 
+            //build a view model //
+            PendingViewModel pvm = new PendingViewModel();
+         
+            IEnumerable<Capture> pendingCaptures = _adminReviewServices.GetUnapprovedCaptures();
+
+            //create a foreach loop that goes thru the list and pulls out images that have NOT been approved. 
+            foreach (Capture capture in pendingCaptures) // looking at comments on line 18 and 23 in Capture.cs // should we display all images since they are the default?
+            {
+               
+                    string mimeType = capture.photo;
+                    List<string> base64 = Convert.ToBase64String(capture.photo);
+                   // return string.Format("fate:{0}; base64,{1}", mimeType, base64);
 
+                      pvm = base64;
+                return View("Pending", pvm);
+
+            }
+            
+            return ViewModels("Pending");
         }
+        // // whould this be followed by a GetEnumerator?
```

Comment:
No, the return statement will end the Action. Using the Enumerator can happen in your cshtml view.